MCLauncher/MainWindow.xaml.cs

```diff
@@ -353,8 +396,7 @@ public class NotifyPropertyChangedBase : INotifyPropertyChanged {
             public event PropertyChangedEventHandler PropertyChanged;
 
             protected void OnPropertyChanged(string name) {
-                if (PropertyChanged != null)
-                    PropertyChanged(this, new PropertyChangedEventArgs(name));
+                PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(name));
```

Comment:
Unrelated change. Also, we use an older .NET version which might not support this.

Reply:
I did not touch the .NET version when building. Do you absolutely want this removed? I'm not making a seperate PR for this.

Reply:
It's unrelated to the objective of the PR, so it shouldn't be here.

Reply:
The code does the same thing, but consider it removed. I apologize for the sloppy PR, this came out of me tweaking this for a friend and I figured it would be nice to give other people the option of different themes.